src/main/java/com/mindsdb/kafka/connect/client/MindsDBClientImpl.java

```diff
@@ -47,12 +47,16 @@ protected PasswordAuthentication getPasswordAuthentication() {
     @Override
     public void createIntegration() throws MindsDBApiException {
         HashMap<String, Object> parameters = new HashMap<>();
-        parameters.put("kafka_host", config.getKafkaHost());
-        parameters.put("kafka_port", config.getKafkaPort());
-        parameters.put("kafka_key", config.getKafkaAuthKey());
-        parameters.put("kafka_secret", config.getKafkaAuthSecret());
+        HashMap<String, Object> connection = new HashMap<>();
+        connection.put("bootstrap_servers", config.getKafkaHost() + ":" + config.getKafkaPort());
+
+        // Need also add checks for 'sasl_mechanism' and 'security_protocol' because
+        // next two params depend on them
+        // see https://kafka-python.readthedocs.io/en/master/apidoc/KafkaClient.html for details
+        connection.put("sasl_plain_username", config.getKafkaAuthKey());
```

Comment:
- Since `sasl_plain_username` and `sasl_plain_password` (key and secret) depend on `security_protocol` and `sasl_mechanism` need to add some checks for them. For e.g it doens't make sence to have `sasl_plain_username` and `sasl_plain_password` without  `security_protocol` and `sasl_mechanism`.
 - See parameters description here: https://kafka-python.readthedocs.io/en/master/apidoc/KafkaClient.html